CoVAR-app/src/app/(pages)/adminTools/components/userList.tsx

```diff
@@ -86,7 +86,7 @@ const UserList = () => {
             return;
         }
 
-        const newRole = user.role === 'VA' ? 'client' : 'VA';
+        const newRole = user.role === 'va' ? 'client' : 'va';
```

Comment:
Are we sure you want to change this back to lowercase?
Will have to inform the rest of the group to recreate their database and will have to update the schema in secrets otherwise e2e is going to fail

Reply:
@eshper  asked me to change it last week. I am not sure what effect it would have on the other stuff.

Reply:
want me to change it back?

Reply:
Just quickly testing something